cmd/build/command.go

```diff
@@ -60,14 +65,15 @@ func Build(ctx context.Context) *cobra.Command {
 			options.CommandArgs = args
 			bc := NewBuildCommand()
 
-			builder, err := bc.getBuilder(options)
-			if err != nil {
+			if err := bc.loadContext(ctx, options); err != nil {
 				return err
 			}
 
-			if err := builder.LoadContext(ctx, options); err != nil {
+			builder, err := bc.getBuilder(options)
+			if err != nil {
```

Comment:
Can we include a comment explaining why this order is important? This is assuming a hidden knowledge (`getBuilder` is reading the manifest and for that, you need to read the manifest first).

I think we need to improve a little bit this command to make it more readable and less error prone. Reading the manifest should be part of the main execution flow and then, pass it to the `getBuilder` function. I think that would make a bit clearer how the command works

Reply:
I agree to move `GetManifest` to that level. I we do that then I see two different options:
- send the error generated in `GetManifest` as a param to `getBuilder` 
- extract logic in `getBuilder` to main execution flow and remove that func.

WDYT? @ifbyol

Reply:
https://github.com/okteto/okteto/issues/3058